game/game.py

```diff
@@ -0,0 +1,29 @@
+import pygame as py
+from lib import server_list
+
+
+class Game:
+    def __init__(self, text, text2):
+        self.text = text
+        self.text2 = text2
+
+    def printer(self):
+        print(self.text + " " + self.text2)
+
+
+if __name__ == "__main__":
+    py.init()
+    window = py.display.set_mode((500, 500))
+    py.display.set_caption("Don\'t u panic asshole")
+
+    run = True
+
+    while run:
+        py.time.delay(100)
```

Comment:
Zamiast funkcji delay() użyć funkcji tick() z obiektu pygame.time.Clock.

Reply:
a to w sumie do wyjebania jest, bo twoja klasa będzie używana xd